sae_lens/toolkit/pretrained_sae_loaders.py

```diff
@@ -627,11 +610,39 @@ def deepseek_r1_sae_loader(
         "sae_lens_training_version": None,
         "activation_fn_str": "relu",
         "normalize_activations": "none",
-        "device": device,
+        "device": options.device,
         "apply_b_dec_to_input": False,
         "finetuning_scaling_factor": False,
     }
 
+
+def deepseek_r1_sae_loader(
+    release: str,
+    sae_id: str,
+    device: str = "cpu",
+    force_download: bool = False,
+    cfg_overrides: Optional[dict[str, Any]] = None,
+) -> tuple[dict[str, Any], dict[str, torch.Tensor], Optional[torch.Tensor]]:
+    """Load a DeepSeek R1 SAE."""
+    # Get repo and file info from pretrained directory
+    sae_directory = get_pretrained_saes_directory()
+    repo_id = sae_directory[release].repo_id
+    filename = sae_directory[release].saes_map[sae_id]
```

Comment:
```suggestion
    repo_id, filename = get_repo_id_and_folder_name(release, sae_id=sae_id)
```

I think the function gets the value for the `path` key in `pretrained_saes.yaml` though, and so we should rename the function to `get_repo_id_and_path` in a separate PR.